docs/code/cloud_storage.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+# pylint: disable=wrong-import-position
+# pylint: disable=wrong-import-order
+# pylint: disable=pointless-string-statement
+# pylint: disable=no-name-in-module
+# pylint: disable=no-value-for-parameter
+# type: ignore[arg-type]
+# flake8: noqa
+
+"""This file includes the python code of auth cloud storage import."""
+
+
+"""Import dataset from cloud platform to the authorized storage dataset"""
+from tensorbay import GAS
+from tensorbay.dataset import AuthData, Dataset
+
+gas = GAS("<YOUR_ACCESSKEY>")
+cloud_client = gas.get_cloud_client()
+
+# Use AuthData to organize a dataset by the "Dataset" class before importing.
+dataset = Dataset("DatasetName")
+
+# TensorBay uses "segment" to separate different parts in a dataset.
+segment = dataset.create_segment()
+
+segment.append(AuthData("train/data/0001.png"))
+segment.append(AuthData("train/data/0002.png"))
+
+dataset_client = gas.upload_dataset(dataset, jobs=8)
+""""""
+
+
+"""Load label file from cloud platform into the AuthData"""
+import json
+
+from tensorbay.label import LabeledBox2D
+
+data = AuthData("train/data/0001.png", target_remote_path="0001.png")
```

Comment:
do we need the cloud storage prefix like `oss://` `s3://` in the cloud path？

Reply:
No need prefix, the cloud config contains such message -- prefix, bucket...